cloud_infrastructure_access/dedicated-understanding-aws.adoc

```diff
@@ -0,0 +1,25 @@
+:context: dedicated-understanding-aws
+[id="welcome-index"]
+= Understanding Cloud Infrastructure Access
+include::modules/common-attributes.adoc[]
+
+Amazon Web Services (AWS) infrastructure access allows
+link:https://access.redhat.com/node/3610411[Customer Portal Organization Administrators]
+and cluster owners to enable AWS Identity and Access Management (IAM) users to
+have federated access to the AWS Management Console for their {product-title}
+cluster.
+
+== Enabling AWS Access
+AWS access can be granted for customer AWS users, and private cluster access can be implemented to suit the needs of your {product-title} environment.
+
+Get started with xref:../cloud_infrastructure_access/dedicated-aws-access.adoc#dedicated-aws-access[Accessing AWS Infrastructure] for your {product-title} cluster. This will guide you through creating an AWS user and account and providing that user with access to the {product-title} AWS account.
+
+Once you have access to the {product-title} AWS account, there are multiple ways to establish private connections to your cluster:
```

Comment:
s/Once/After

> ! once adv
Use only to mean “one time.” Do not use as a conjunction to mean “after” or “when.”

> ! there is, there are v
Avoid at the beginning of an independent clause. In most cases rewrite to remove the phrase.

maybe s/there are multiple ways to establish private connections to your cluster:/use one or more of the following methods to establish a private connection to your cluster: